### PR TITLE
Release Notes - minor update

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds a section called "Upgrade Notes" in the 3.1 release notes. A statement pertaining to MaaS clusters has been added.

CleanShot 2023-01-04 at 07 54 34

### CODE REVIEW

**Comment by antongisli** [comment]
imo we shoudl reference or add details about:
- why does editing prekubeadm files cause repaves?
- what is prekubeadm?

**Review by resmaa — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1078#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1078#pullrequestreview-1236049097" title="Approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by karl-cardenas-coding** [comment]
> * why does editing prekubeadm files cause repaves?
> * what is prekubeadm?

Explaining kubeadm is outside of the scope of release notes. But I see your point and agree. Instead, let's call it initialization and state that the memory swap is disabled. That explains the change, and then we can state that applying the change causes replacement.

**Pull request merged**
